### DISCUSSION TITLE
Add unique index on notification_tokens token?

### DISCUSSION THREAD

**Discussion by sedubois** [excid3/noticed#292] [Ideas]
After installing Noticed for FCM push notifications, the schema looks as below. Shouldn't there be a unique index on token? Or maybe on user_id + platform + token?

```ruby
  create_table "notification_tokens", force: :cascade do |t|
    t.bigint "user_id", null: false
    t.string "platform"
    t.string "token"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["user_id"], name: "index_notification_tokens_on_user_id"
  end
```

**Comment by excid3** [comment]
Noticed doesn't provide a `notification_tokens` table. Jumpstart Pro does if you meant to post this in that repository.

**Reply by sedubois** [reply]
I've opened a PR: https://github.com/jumpstart-pro/jumpstart-pro-android/pull/293
